flutter/linux/wayland_shortcuts_inhibit.cc

```
@@ -0,0 +1,232 @@
+    // Mark as connected to prevent duplicate connections
+    g_object_set_data(G_OBJECT(widget), kFocusHandlersConnectedKey, GINT_TO_POINTER(1));
+
+    // If window already has focus, create inhibitor now
+    if (gtk_window_has_toplevel_focus(GTK_WINDOW(widget))) {
+      inhibit_keyboard_shortcuts(GTK_WINDOW(widget));
+    }
+  }
+}
+
+// Public API: Initialize shortcuts inhibit for a sub-window
+void wayland_shortcuts_inhibit_init_for_subwindow(void* view) {
```

Cppcheck (2.19.0)

[style] The function 'wayland_shortcuts_inhibit_init_for_subwindow' is never used.

(unusedFunction)